app/server/server.ts

```diff
@@ -48,8 +48,18 @@ server.use(routes.productsProvider("/api/"));
 server.use(
   "/consent/",
   (req, res, next) => {
-    // we want this route to be loaded in an iframe
+    // This route can be loaded in an iframe from the domains listed below only
+    const allowedDomains = [
```

Comment:
You can probably build a single URL from the conf.DOMAIN constant, that way were not mentioning gulocal.com in the headers of all responses